frontend/src/components/sessions/data-visualization/StatisticsSection.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { Flex, Grid, Text } from "@chakra-ui/react";
+
+import StatisticCard, { type StatisticCardVariant } from "./StatisticCard";
+
+export type StatisticsConfig = {
+  [key: string]: {
+    title: string;
+    formatValue: (value: number) => string;
+    variant?: StatisticCardVariant;
+    fallbackValue?: string;
+  };
+};
+
+type Statistics<T> = {
+  [key in keyof T]?: number;
+};
+
+const getKeys = <T,>(obj: StatisticsConfig) =>
+  Object.keys(obj) as (keyof T & string)[];
+
+export const NUMBER_FORMAT = (value: number) => value.toFixed(0);
+export const PERCENTAGE_FORMAT = (value: number) => `${NUMBER_FORMAT(value)}%`;
+export const NTH_FORMAT = (value: number) => {
+  const lastDigit = value % 10;
+  let suffix = "th";
+  if (lastDigit === 1) suffix = "st";
+  if (lastDigit === 2) suffix = "nd";
+  if (lastDigit === 3) suffix = "rd";
+  return `${NUMBER_FORMAT(value)}${suffix}`;
+};
+
+const StatisticsSection = <T,>({
+  config,
+  values,
+}: {
+  config: StatisticsConfig;
+  values: Statistics<T>;
+}) => (
+  <Flex direction="column" gap={2}>
+    <Text as="h2" color="grey.300" textStyle="eyebrow">
```

Comment:
veryyyyyyy nitty nit: can we align the statistics card to the header? :D i think they're slightly misaligned

<img width="489" alt="Screenshot 2023-05-29 at 12 27 44 PM" src="https://github.com/uwblueprint/jump-math/assets/41273486/e2c8a77b-7f0b-4a61-b30f-bb698a84b13d">

Reply:
oops, not a nitty nit, that's just a mistake LMAO